analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -30,6 +30,23 @@ import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
 
 class MavenSupportTest : WordSpec({
+    "getOriginalScm()" should {
+        "return the parent's SCM if the child SCM uses the parent's SCM URL as a prefix" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    url = "https://github.com/spring-projects/spring-boot/childArtifactName"
+                }
+                parent = MavenProject().apply {
+                    scm = Scm().apply {
+                        url = "https://github.com/spring-projects/spring-boot"
+                    }
+                }
+            }
+
+            MavenSupport.getOriginalScm(mavenProject) shouldBe mavenProject.parent.scm
```

Comment:
Do you really want to assert instances here? For readablity anyway it would be easier to only assert the URL as a string and spell out the expected value explicitly.